taf/plugins/pytest_sut_monitor.py

```diff
@@ -359,7 +368,7 @@ def monitor_init(self, env_init):
         return self.sut_monitor
 
     @pytest.fixture(scope=setup_scope(), autouse=True)
-    def monitor(self, request, env_main, monitor_init):
+    def monitor(self, request, env_main, monitor_init):  # pylint: disable=W0613
```

Comment:
err it's not in the fatal list, so pylint won't block on W0613.  So it is acting just as a warning currently.

https://github.com/taf3/taf/blob/ci/project_checker.py#L958